README.md

```diff
@@ -86,6 +86,15 @@ To build ML.NET from source please visit our [developers guide](docs/project-doc
 |**Windows x86**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobName=Windows_x86_NetCoreApp21&configuration=Windows_x86_NetCoreApp21%20Debug_Build)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=104&branchName=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobName=Windows_x86_NetCoreApp21&configuration=Windows_x86_NetCoreApp21%20Release_Build)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=104&branchName=master)|
 |**Windows NetCore3.0**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobName=Windows_x64_NetCoreApp30&configuration=Windows_x64_NetCoreApp30%20Debug_Build)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=104&branchName=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/machinelearning/MachineLearning-CI?branchName=master&jobName=Windows_x64_NetCoreApp30&configuration=Windows_x64_NetCoreApp30%20Release_Build)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=104&branchName=master)|
 
+## Release process
+
+ML.NET has four kinds of releases: nightly builds, monthly preview releases, periodic general availability (GA) releases, and fix releases. We detail each kind of release below.
+
+1. **Nightly builds:** these are supported in NuGet, and support the most cutting-edge work. The trade-off is that these releases are not as rigorously tested.
```

Comment:
@antoniovs1029 Is there a link I could link people to for this line? Not too familiar with NuGet, but I would want it to be a link that users could follow in order to get the nightly build.

Reply:
Hi. Currently there's no public feed for the users to get NuGets from the nightly builds. I am working on this with @safern and hopefully we will be able to provide you with a link for this by the end of the week.

Reply:
https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=MachineLearning